internal/test/e2e/gouging_test.go

```diff
@@ -187,7 +187,7 @@ func TestAccountFunding(t *testing.T) {
 	time.Sleep(defaultHostSettings.PriceTableValidity)
 
 	// fund the account again
-	tt.OK(w.RHPFund(context.Background(), c.ID, c.HostKey, c.HostIP, c.SiamuxAddr, types.Siacoins(1)))
+	tt.OK(w.RHPFund(context.Background(), c.ID, c.HostKey, c.HostIP, c.SiamuxAddr, types.Siacoins(1).Div64(2)))
```

Comment:
Seems like this is related to no longer handling exceeding the max balance?

Reply:
Leftover I think from the branch at one point in time. Reverting the change.